MAVRICK-1/e-commerce_website
# Compare page and it's functionality Fixed

Compare page and it's functionality Fixed

## Description

no compare page. the Compare button is one of the top features on a website features list. page and functonality added.

## Issue
[Issue Link](https://github.com/MAVRICK-1/e-commerce_website/issues/150) resolve #150 

## How Has This Been Tested?

- go to website and login
- view products and add to compare.
- now check that product difference in compare page.

## Types of changes




## Checklist:
